data/data-pipeline/data_pipeline/etl/sources/national_risk_index/etl.py

```diff
@@ -93,33 +96,92 @@ def transform(self) -> None:
+                NRI_TRACT_COL: TRACT_COL,
                 self.RISK_INDEX_EXPECTED_ANNUAL_LOSS_SCORE_INPUT_FIELD_NAME: self.RISK_INDEX_EXPECTED_ANNUAL_LOSS_SCORE_FIELD_NAME,
             },
             inplace=True,
         )
 
+        # Only use disasters linked to climate change
+        disaster_categories = [
+            "AVLN",  # Avalanche
+            "CFLD",  # Coastal Flooding
+            "CWAV",  # Cold Wave
+            "DRGT",  # Drought
+            "HAIL",  # Hail
+            "HWAV",  # Heat Wave
+            "HRCN",  # Hurricane
+            "ISTM",  # Ice Storm
+            "LNDS",  # Landslide
+            "RFLD",  # Riverine Flooding
+            "SWND",  # Strong Wind
+            "TRND",  # Tornado
+            "WFIR",  # Wildfire
+            "WNTW",  # Winter Weather
+        ]
+
+        # Some disaster categories do not have agriculture value column
+        agri_columns = [str(x) + "_EALA" for x in disaster_categories if str(x) + "_EALA" in list(df_nri.columns)]
+
+        population_columns = [str(x) + "_EALP" for x 
+            in disaster_categories if str(x) + "_EALP" in list(df_nri.columns)]
+
+
+        buildings_columns = [str(x) + "_EALB" for x 
+            in disaster_categories if str(x) + "_EALB" in list(df_nri.columns)]
+
+        disaster_population_sum_series = df_nri[
+            population_columns
+        ].sum(axis=1)
+
+        disaster_agriculture_sum_series = df_nri[
+            agri_columns
+        ].sum(axis=1)
+
+        disaster_building_sum_series = df_nri[
+            buildings_columns
+        ].sum(axis=1)   
+
+
         # Population EAL Rate = Eal Valp / Population
         df_nri[self.EXPECTED_POPULATION_LOSS_RATE_FIELD_NAME] = (
-            df_nri[self.EXPECTED_ANNUAL_LOSS_POPULATION_VALUE_INPUT_FIELD_NAME]
-            / df_nri[self.POPULATION_INPUT_FIELD_NAME]
+            disaster_population_sum_series / df_nri[self.POPULATION_INPUT_FIELD_NAME]
         )
 
         # Agriculture EAL Rate = Eal Vala / Agrivalue
         df_nri[self.EXPECTED_AGRICULTURE_LOSS_RATE_FIELD_NAME] = (
-            df_nri[
-                self.EXPECTED_ANNUAL_LOSS_AGRICULTURAL_VALUE_INPUT_FIELD_NAME
-            ]
-            / df_nri[self.AGRICULTURAL_VALUE_INPUT_FIELD_NAME]
+            disaster_agriculture_sum_series / df_nri[self.AGRICULTURAL_VALUE_INPUT_FIELD_NAME]
         )
 
         # divide EAL_VALB (Expected Annual Loss - Building Value) by BUILDVALUE (Building Value ($)).
         df_nri[self.EXPECTED_BUILDING_LOSS_RATE_FIELD_NAME] = (
-            df_nri[self.EXPECTED_ANNUAL_LOSS_BUILDING_VALUE_INPUT_FIELD_NAME]
-            / df_nri[self.BUILDING_VALUE_INPUT_FIELD_NAME]
+            disaster_building_sum_series / df_nri[self.BUILDING_VALUE_INPUT_FIELD_NAME]
+        )
+
+        # Reduce columns.
+        # Note: normally we wait until writing to CSV for this step, but since the file is so huge,
+        # move this up here for performance reasons.
+        df_nri = df_nri[
```

Comment:
Yeah, this also looks like old code, making me think you branched off an older version of this repo. This is no longer necessary now that we're at the tract level.